src/raw_iter.rs

```diff
@@ -62,31 +62,70 @@ where
     }
 }
 
-enum DynamicRawColors<'a> {
+/// Iterator over dynamic color. Positions are dependent on the row order.
+#[allow(missing_debug_implementations)]
```

Comment:
You've added a manual implementation of `Debug` and I think that this shouldn't be necessary anymore.
```suggestion
```